### PR TITLE
fix: add bottom margin overflow in hastags of event-cards

### LINKED ISSUE #2346: Add bottom margin to event tags in event card
**Describe the bug**

The tags in event cards shown on homepage are quite close to the bottom boundary of the card. It will be better to add a bottom margin.
Also in the responsive mode, the tags go out of cards.
**To Reproduce**
1. Go to '/'
2. See error

**Expected behavior**

There should be some gap between tags and bottom boundary of card
**Screenshots**

Screenshot from 2019-03-20 15-47-08

Screenshot from 2019-03-20 16-12-58

**Additional context**

I am working on it.

### PR DESCRIPTION
**Pull request by ShreyanshSrivastava**
Fixes #2346 

#### Short description of what this resolves:
The proposed change in code will add a bottom margin as well as prevent the overflow of hashtags from the even cards.

#### Changes proposed in this pull request:

- Instead of fixing the height of the card, make it auto to adjust it according to the size of the contents inside the card.

### **Old View**
old-view
### **New View**
new-view

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Now, it can create layout issues in grid.

**Comment by iamareebjamal** [comment]
image

**Comment by ShreyanshSrivastava** [comment]
> Now, it can create layout issues in the grid.
@iamareebjamal Sir please check now, I have proposed another way which will not disturb the layout as well as prevent hashtags from flowing out.

**Comment by ShreyanshSrivastava** [comment]
view
This is preventing the hashtags from overflow as well as not disturbing the grid layout

File: app/styles/partials/event-card.scss [hunk 1]
```diff
@@ -17,6 +17,20 @@
   }
 }
 
+.extra.content {
+
+  span {
+
+    &:last-child {
```
Comment (iamareebjamal) [inline app/styles/partials/event-card.scss:24]: Just use a class, no need for a complicated selector

Comment (ShreyanshSrivastava) [reply app/styles/partials/event-card.scss]: @iamareebjamal Done sir, please check now.

**Comment by iamareebjamal** [comment]
Keep PR titles short, to the point and according to the format

**Comment by ShreyanshSrivastava** [comment]
> Keep PR titles short, to the point and according to the format

Yes, sir I will keep that in mind from next time

**Pull request merged**